scarpe-components/lib/scarpe/components/calzini/art_widgets.rb

```diff
@@ -7,7 +7,7 @@ def arc_element(props, &block)
     HTML.render do |h|
       h.div(id: html_id, style: arc_style(props)) do
         h.svg(width: props["width"], height: props["height"]) do
-          h.path(d: arc_path(props), transform: "rotate(#{rotate}, #{props["width"] / 2}, #{props["height"] / 2})")
+          h.path(d: arc_path(props), transform: "rotate(#{rotate}, #{props["width"] / 2}, #{props["height"] / 2})","transform-box": "fill-box","transform-origin": "50% 50%")
```

Comment:
Is changing the transform-origin going to change all existing rotations on arc, etc?

Reply:
That might be fine, but is this the right one?

Reply:
no transform-origin is for something else, i fogot , for cap its only for line i think

Reply:
transform origin is for some property where svg should be spinned based on center or corner  i forgot what it is for in shoes i will look and get back in a sec

Reply:
yeah here for this one we need transform-origin but rotating based on center or corner is weird in svgs so i still had to look into them 

https://github.com/gintama91/scarpe/blob/444e080e29cb20d133181292b854524531fc3923/docs/static/manual.md?plain=1#L1852

Reply:
in some article i read we now dont need to think about matrixes and stuff but i still dont really understand if it even changes to center or curve coz arc is just practically disappearing slowly. (i am probably doing something wrong.)

Reply:
wait sorry i just realised what you asked is completely different , sorry

yeah i need to get it from drawing context and update to either center or corner